### PR TITLE
Implementing prerequisites for NuGet publishing

### PR DESCRIPTION
**Pull request by Nockiro**
This PR contains the following changes:
- Creating solution and project file to wrap the controls and additional classes
- Minor code cleanup
- Adding configuration to publish controls as NuGet package

The NuGet Details in VS would show the following information with this PR:
image

### CODE REVIEW

File: StylableWinFormsControls/StylableWinFormsControls/Controls/StylableLabel.cs [hunk 1]
```diff
@@ -87,7 +83,7 @@ internal virtual TextFormatFlags CreateTextFormatFlags(Size constrainingSize)
     /// Source: MeasureTextCache in System.Windows.Forms.Layout.LayoutUtils (part of it)
     /// Cache mechanism added for VSWhidbey 500516
     /// 3000 character strings take 9 seconds to load the form
-    internal class MeasureTextCache
+    public class MeasureTextCache
```
Comment (wolframhaussig) [inline StylableWinFormsControls/StylableWinFormsControls/Controls/StylableLabel.cs:86]: Maybe we should extract this class to be in a separate File? This feature can be useful wherever we want to measure the width of a string

Comment (wolframhaussig) [reply StylableWinFormsControls/StylableWinFormsControls/Controls/StylableLabel.cs:86]: and maybe rename it? It does not cache anything so the naming is a bit weird...

**Review by wolframhaussig — APPROVED** [review]
LGTM, just one minor proposal

File: StylableWinFormsControls/StylableWinFormsControls/Controls/StylableLabel.cs (hunk 1 above)
Comment (Nockiro) [reply StylableWinFormsControls/StylableWinFormsControls/Controls/StylableLabel.cs]: I've extracted the internal classes into separate files now and used the InvalidateCache method from the MeasureTextCache, that one was simply not connected 😄

**Review by wolframhaussig — APPROVED** [review]

**Pull request merged**